panther_docking/src/panther_charging_dock.cpp

```diff
@@ -90,6 +96,12 @@ void PantherChargingDock::declareParameters(const rclcpp_lifecycle::LifecycleNod
 
   nav2_util::declare_parameter_if_not_declared(
     node, name_ + ".filter_coef", rclcpp::ParameterValue(0.1));
+
+  nav2_util::declare_parameter_if_not_declared(
+    node, name_ + ".use_wibotic_info", rclcpp::ParameterValue(true));
+
+  nav2_util::declare_parameter_if_not_declared(
+    node, name_ + ".wibotic_info_timeout", rclcpp::ParameterValue(1.5));
```

Comment:
In readme default is 0.2.

Reply:
Changing in readme. The wibotic rate is 1s